### DISCUSSION TITLE
Recommended way of incorporating tailwindcss into create-react-app

### DISCUSSION THREAD

**Discussion by creeefs** [tailwindlabs/tailwindcss#1486] [General]
Hello,

Is there a recommended way of incorporating `tailwindcss` into [create-react-app](https://create-react-app.dev/)? Given CRA's popularity, I feel like it could have the dual benefit of saving people time on config (i.e. start using tailwind faster!) as well as reduce the number of config related questions by establishing a stronger convention.

Thanks!
Chris

**Comment by adamwathan** [comment]
Unfortunately create react app doesn’t support a custom PostCSS configuration but hopefully they will accept this PR:

https://github.com/facebook/create-react-app/pull/8474

Until then you can use tools like craco or build your CSS using a separate build script outside the normal CRA process. Sucks :/

**Comment by creeefs** [comment]
Thanks for the reply! I gave that CRA PR a 👍, so hopefully that does get merged in the near future 😄 

If you think it's worth adding what you said above somewhere in the documentation, I'd be happy to take a stab and open a PR.
